### PR TITLE
Create SP19-Fake News.md

### PR DESCRIPTION
**Pull request by nicknytko**

### CODE REVIEW

File: _projects/SP19-Fake News.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: project
+title: Fake News
+image: <link to project image (optional)>
```
Comment (ShanTulshi) [inline _projects/SP19-Fake News.md]:
```suggestion
```

File: _projects/SP19-Fake News.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+desc: |
+ Slice fake news articles for fun and for profit.
+github: <github link (optional)>
```
Comment (ShanTulshi) [inline _projects/SP19-Fake News.md]:
```suggestion
```

File: _projects/SP19-Fake News.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+dl: https://drive.google.com/open?id=1HlgbFNRmI1L1uYKqxMwQ3baDfqM6HIMx
```
Comment (ShanTulshi) [inline _projects/SP19-Fake News.md]: Make sure this link is accessible by anyone with the link

File: _projects/SP19-Fake News.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+semester: Spring 2019
+team: 25
+youtube: lp4a7KxF64g
```
Comment (ShanTulshi) [inline _projects/SP19-Fake News.md]: Video is unavailable

Comment (nicknytko) [reply _projects/SP19-Fake News.md]: Weird the video is working for me (https://www.youtube.com/watch?v=lp4a7KxF64g), what's showing up for you?

Comment (ShanTulshi) [reply _projects/SP19-Fake News.md]: It showed up as unavailable, but it seems to work now

**Pull request merged**